dist/js/styleguide.js

```diff
@@ -194,15 +193,17 @@ $(function(){
   });
 
   window.Identity = Identity;
-});
+})();
 
-$(function(){
+(function(){
 
   var Identity = window.Identity || {};
 
   Identity.forms = {};
-
+  var initialized = false;
```

Comment:
Why do we need this? If we don't want to initialize module twice we could check if Identity.forms.toggleShowPassword present.

Reply:
That's a great idea, I'll update